ci-operator/step-registry/openshift-qe/installer/bm/self-sched/openshift-qe-installer-bm-self-sched-ref.yaml

```diff
@@ -0,0 +1,32 @@
+ref:
+  as: openshift-qe-installer-bm-self-sched
+  from_image:
+    namespace: ci
+    name: telco-bastion
+    tag: latest
+  commands: openshift-qe-installer-bm-self-sched-commands.sh
+  credentials:
+    - namespace: ci # this entry injects the custom credential
+      name: cluster-secrets-metal-perfscale-selfsched
+      mount_path: /secret
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+    - name: LAB
+      default: "scalelab-stage"
+      documentation: |-
+        Lab where the hardware is located. Either performancelab, scalelab or scalelab-stage.
+    - name: MODEL
+      default: "FC640"
+      documentation: |-
+        Hardware model type
```

Comment:
should we add example models here to make it clear what can be accepted?

Reply:
Sure let me add some examples

Reply:
Added some examples, complete filtering doc can be found here: https://github.com/redhat-performance/quads/blob/latest/docs/quads-host-metadata-search.md